IPython/terminal/ptutils.py

```diff
@@ -10,7 +10,7 @@
 import unicodedata
 from wcwidth import wcwidth
 
-from IPython.core.completer import IPCompleter, provisionalcompleter, rectify_completions, cursor_to_position
+from IPython.core.completer import IPCompleter, provisionalcompleter, rectify_completions, cursor_to_position, _deduplicate_completions
```

Comment:
Line a bit too long? We used to keep line lengths ~80, though I don't know if that's being followed still :)